src/main/java/com/coveros/selenified/Browser.java

```diff
@@ -134,19 +134,23 @@ public String getScreensize() {
 
     public String getDetails() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS3776)